spec/motion-spec/matchers/respond_to_spec.rb

```diff
@@ -0,0 +1,48 @@
+# -*- encoding : utf-8 -*-
+describe 'Matcher::RespondTo' do
+  class TestCase
+    def call_me(a, b, c)
```

Comment:
Unused method argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used. You can also write as `call_me(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used. You can also write as `call_me(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `c`. If it's necessary, use `_` or `_c` as an argument name to indicate that it won't be used. You can also write as `call_me(*)` if you want the method to accept any arguments but don't care about them.